<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginDlg</class>
 <widget class="QDialog" name="LoginDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>778</width>
    <height>476</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LoginDlg</string>
  </property>
  <widget class="QPushButton" name="LoginButton">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>390</y>
     <width>150</width>
     <height>46</height>
    </rect>
   </property>
   <property name="text">
    <string>OK!</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ExitButton">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>390</y>
     <width>150</width>
     <height>46</height>
    </rect>
   </property>
   <property name="text">
    <string>CANCEL!</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>110</y>
     <width>108</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>账号</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>220</y>
     <width>108</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>密码</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="accountEdit">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>90</y>
     <width>401</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="passwordEdit">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>200</y>
     <width>401</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>LoginButton</sender>
   <signal>clicked()</signal>
   <receiver>LoginDlg</receiver>
   <slot>ClickLogin()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>484</x>
     <y>412</y>
    </hint>
    <hint type="destinationlabel">
     <x>388</x>
     <y>237</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ExitButton</sender>
   <signal>clicked()</signal>
   <receiver>LoginDlg</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>684</x>
     <y>397</y>
    </hint>
    <hint type="destinationlabel">
     <x>678</x>
     <y>316</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>ClickLogin()</slot>
 </slots>
</ui>
